sdk/schemaregistry/Azure.Data.SchemaRegistry/README.md

```diff
@@ -0,0 +1,157 @@
+
+The Azure Schema Registry service allows developers to provide and retrieve data schemas from a centralized repository for use in messaging systems.
+
+## Getting started
+
+### Install the package
+
+Install the Azure Schema Registry client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+Install-Package Azure.Data.SchemaRegistry
+```
+
+### Prerequisites
+
+* An [Azure subscription][azure_sub]
+* An [Event Hubs namespace][event_hubs_namespace]
+
+If you need to [create an Event Hubs namespace][create_event_hubs_namespace], you can use the Azure Portal or [Azure PowerShell][azure_powershell].
+
+You can use Azure PowerShell to create the Event Hubs namespace with the following command:
+
+```PowerShell
+New-AzEventHubNamespace -ResourceGroupName myResourceGroup -NamespaceName namespace_name -Location eastus
+```
+
+### Authenticate the client
+
+In order to interact with the Azure Schema Registry service, you'll need to create an instance of the [Schema Registry Client][schema_registry_client] class. To create this client, you'll need Azure resource credentials and the Event Hubs namespace hostname.
+
+#### Get credentials
+
+To acquire authenicated credentials and start interacting with Azure resources, please see the [quickstart guide here][quickstart_guide].
+
+#### Get Event Hubs namespace hostname
+
+The simpliest way is to use the [Azure portal][azure_portal] and navigate to your Event Hubs namespace. From the Overview tab, you'll see `Host name`. Copy the value from this field.
+
+#### Create SchemaRegistryClient
+
+Once you have the Azure resource credentials and the Event Hubs namespace hostname, you can create the [SchemaRegistryClient][schema_registry_client]. You'll also need the [Azure.Identity][azure_identity] package to create the credential.
+
+```C# Snippet:CreateSchemaRegistryClient
+string endpoint = "<event_hubs_namespace_hostname>";
+var credentials = new ClientSecretCredential(
+    "<tenant_id>",
+    "<client_id>",
+    "<client_secret>"
+);
+var client = new SchemaRegistryClient(endpoint, credentials);
+```
+
+## Key concepts
+
+### Schemas
+
+A schema has 6 components:
+- Group Name
+- Schema Name
+- Schema ID
+- Serialization Type
+- Schema Content
+- Schema Version
+
+These components play different roles. Some are used as input into the operations and some are outputs. Currently, [SchemaProperties][schema_properties] only exposes those properties that are potential outputs that are used in SchemaRegistry operations. Those exposed properties are `Content` and `Id`.
+
+## Examples
+
+The following shows examples of what is available through the SchemaRegistryClient. There are both sync and async methods available for these client operations.
+
+* [Register a schema](#register-a-schema)
+* [Retrieve a schema ID](#retrieve-a-schema-id)
+* [Retrieve a schema](#retrieve-a-schema)
+
+### Register a schema
+
+Register a schema to be stored in the Azure Schema Registry.
+
+```C# Snippet:RegisterSchema
```

Comment:
This ends up rendering pretty strangely in the README.

Reply:
Fixed.